lessons/08-html-site/contacts.html

```diff
@@ -0,0 +1,54 @@
+<html lang="en">
+<head>
+    <style>
+        * {
+            font-family: monospace;
+        }
+
+        header h2 {
+            text-align: center;
+            font-size: 30px;
+        }
+
+        body {
+            background-color: papayawhip;
+        }
+    </style>
+    <title>Contacts</title>
+</head>
+<header>
+    <img src="/Users/greanncyramales/Code/comit-react/lessons/08-html-site/myfacelol/UNADJUSTEDNONRAW_thumb_2e.jpg"
```

Comment:
Instead of doing an absolute URL like this, you should reference it relatively. e.g. `src="myfacelol/UNADJUSTEDNONRAW_thumb_2e.jpg"`